libipl/p10/ipl0.C

```diff
@@ -806,7 +806,10 @@ static int ipl_proc_clock_test(void)
 		if (!ipl_is_functional(proc))
 			continue;
 
-		ipl_log(IPL_INFO,"Running p10_clock_test HWP on processor %d\n",
+		if (!ipl_is_master_proc(proc))
+			continue;
+
+		ipl_log(IPL_INFO,"Running p10_clock_test HWP on primary processor %d\n",
```

Comment:
Comment Same as previous commit .

Reply:
done, now added new method to get functional master proc rather than loop 2 times